### PR TITLE
SEO-173296 WPF Chart Area

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @MallikaRK ,

Have renamed image folder, changed and image alt.

Regards,
Yvone.

### CODE REVIEW

**Review by MallikaRK — APPROVED** [review]
Approved

File: wpf/Charts/Area.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ chart.AreaBorderThickness = new Thickness(3);
 
 {% endtabs %}
 
-![Customizing Plot Area in WPF Chart](Area_images/wpf-chart-plot-area-customization.jpeg)
+![Customizing plot area in WPF Chart](area_images/wpf-chart-plot-area-customization.jpeg)
```
Comment (Girija-Rajendran) [inline wpf/Charts/Area.md:66]: Changed in folder?
image

Comment (Yvone-Atieno) [reply wpf/Charts/Area.md:66]: Yes the folder is changed

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**